TheAurorian/src/main/resources/assets/theaurorian/patchouli_books/the_aurorian_guide/zh_cn/entries/aurorianbasics/5lootscrapping.json

```
@@ -0,0 +1,41 @@
+{
+    "name": "碎片化",
+    "icon": "theaurorian:auroriansteel",
+    "category": "1aurorianbasics",
+    "sortnum": "5",
+    "readbydefault": true,
+    "pages": [
+        {
+            "type": "spotlight",
+            "title": "碎片化",
+            "item": "theaurorian:auroriansteel",
+            "text": "极光幽境的许多材料都可以被熔炼为碎片。碎片可以被用于修复工具、制作其它种类的工具以及合成一种特殊的钢：$(#008cff)极光钢$()！你也可以使用粉碎器以获得双倍的碎片！"
+        },
+        {
+            "type": "smelting",
+            "recipe": "theaurorian:aurorianitesword",
+            "recipe2": "theaurorian:crystallineshield"
+        },
+        {
+            "type": "crafting",
+            "recipe2": "theaurorian:items/aurorianiteingot",
+            "text": "所有种类的碎片都可被转为它们对应的锭。"
+        },
+        {
+            "type": "crafting",
+            "recipe": "theaurorian:items/crystallineingot",
+            "recipe2": "theaurorian:items/umbraingot"
+        },
+        {
+            "type": "crafting",
+            "recipe": "theaurorian:items/auroriansteel",
+            "text": "$(#008cff)极光钢$()是极光幽境中最强大的材料，它可以被制成工具和盔甲。在$(#008cff)魔法物品$()部分中查看更多的信息。"
```

```suggestion
            "text": "$(#008cff)极光钢$()是极光幽境中最强大的材料，它可以被制成工具和盔甲。你可以在$(#008cff)魔法物品$()部分中查看更多的信息。"

```
时刻注意动词原形打头的句子，这种句子作为祈使句会省略掉人称，所以翻译时一定要补足，而且根据句子的需要，可以将祈使语气翻译或者较弱地翻译出来

---

要改的多了，麻烦帮忙搜索一下，关键词应该就`部分`和`章节`